GVRf/Sample/gvrtextviewsample/src/org/gearvrf/gvrtextviewsample/SampleViewManager.java

```diff
@@ -32,8 +36,14 @@
             "veryverygood", "veryverygood", "veryveryverygood" };
     private final int[] colors = new int[] { Color.RED, Color.YELLOW,
             Color.GREEN, Color.WHITE, Color.MAGENTA };
+    private final IntervalFrequency[] frequencies = new IntervalFrequency[] {
+            IntervalFrequency.HIGH, IntervalFrequency.MEDIUM,
+            IntervalFrequency.LOW };
     private float textSize;
     private int counter = 0;
+    Random random = new Random();
+
+    ArrayList<GVRTextViewSceneObject> textviews = new ArrayList<GVRTextViewSceneObject>();
 
```

Comment:
LinearLayout mTextView; 
Can be deleted to avoid distraction. Seems there because of some historic reason…